lib/widgets/widgets/iframe_embed/iframe_embed_widget.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:cogboardmobileapp/models/widget_model.dart';
+import 'package:cogboardmobileapp/translations/app_localizations.dart';
+import 'package:flutter/material.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class IframeEmbedWidget extends StatelessWidget {
+  final DashboardWidget widget;
+
+  IframeEmbedWidget({
+    this.widget,
+  });
+
+  String get iframeUrl {
+    return widget.content['iframeUrl'];
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Text(
+        '<iframe/>',
```

Comment:
🥇 [NOW] give some better text here like: `Iframe preview not supported`

Reply:
done